doc/guides/west/release-notes.rst

```diff
@@ -1,6 +1,59 @@
 West Release Notes
 ##################
 
+v0.8.0
+******
+
+This is a feature release which changes the manifest schema by adding support
+for a ``path-prefix:`` key in an ``import:`` mapping, along with some other
+features and fixes.
+
+- Manifest import mappings now support a ``path-prefix:`` key, which foo places
+  the project and its imported repositories in a subdirectory of the workspace.
+  See :ref:`west-manifest-ex3.3` for an example.
+- The west command line application can now also be run using ``python3 -m
+  west``. This makes it easier to run west under a particular Python
+  interpreter without modifying the :envvar:`PATH` environment variable.
+- :ref:`west manifest --path <west-manifest-path>` prints the absolute path to
+  west.yml
+- ``west init`` now supports an ``--mf foo.yml`` option, which initializes the
```

Comment:
Is the option for this really `--mf`?

Reply:
> Is the option for this really `--mf`?

Yep. West's argument parsing treats anything with two or more characters is a long option.